backends/bmv2/run-bmv2-ptf-test.py

```diff
@@ -1,16 +1,16 @@
 #!/usr/bin/env python3
 
 import argparse
-import subprocess
 import os
 import sys
-import socket
 import random
+import logging
```

Comment:
I will let the owners of this file review the changes

Reply:
That's use! We approve :)